### PR TITLE
add getLengthWithoutGroupHeaders method

### PR DESCRIPTION
**Pull request by bcherny**

### CODE REVIEW

File: slick.dataview.js [hunk 1]
```diff
@@ -379,6 +379,11 @@
       return rows.length;
     }
 
+    // (void) => Number
+    function getLengthWithoutGroupHeaders() {
```
Comment (albertywu) [inline slick.dataview.js:383]: also see `getFilteredItems`. This may be what you want (excludes filtered items and grouped rows).

**Comment by bcherny** [comment]
good catch, @albertywu! closing.

**Comment by bcherny** [comment]
nevermind, reopening. `getFilteredItems` doesn't take group collapsing into account!

**Comment by albertywu** [comment]
ahh good catch - :+1:

**Pull request merged**